src/components/modals/modal-wrapper.tsx

```diff
@@ -72,55 +73,55 @@ export const ModalWrapper = ({
       return null;
     }
     switch (type) {
```

Comment:
Please fix the `type` type. It should by ModalType enum

Reply:
Types fixed